### PR TITLE
Report remaining chars

### PR DESCRIPTION
**Pull request by nvdaes**

### CODE REVIEW

**Comment by nvdaes** [comment]
Fixes issue #2267

File: src/routes/_components/LengthIndicator.html [hunk 1]
```diff
@@ -42,11 +41,11 @@
...
       lengthToDisplay: ({ length, max }) => max - length,
-      lengthLabel: ({ overLimit, lengthToDisplayDeferred }) => {
-        if (overLimit) {
-          return formatIntl('intl.overLimit', { count: -lengthToDisplayDeferred })
+      lengthVerbosity: ({ lengthToDisplayDeferred }) => {
+        if (lengthToDisplayDeferred > 10) {
+          return 'off'
         } else {
-          return formatIntl('intl.underLimit', { count: lengthToDisplayDeferred })
+          return 'polite'
```
Comment (nolanlawson) [inline src/routes/_components/LengthIndicator.html:48]: Now that `intl.overLimit` and `intl.underLimit` are unused, we can remove them from `en-US.js` (and the other intl files in the same directory).

File: src/routes/_components/LengthIndicator.html [hunk 2]
```diff
@@ -42,11 +41,11 @@
     store: () => store,
     computed: {
       lengthToDisplay: ({ length, max }) => max - length,
-      lengthLabel: ({ overLimit, lengthToDisplayDeferred }) => {
-        if (overLimit) {
-          return formatIntl('intl.overLimit', { count: -lengthToDisplayDeferred })
+      lengthVerbosity: ({ lengthToDisplayDeferred }) => {
```
Comment (nolanlawson) [inline src/routes/_components/LengthIndicator.html:44]:
```suggestion
      lengthVerbosity: ({ lengthToDisplayDeferred }) => {
        // When approaching the limit, notify screen reader users
```

**Review by nolanlawson — APPROVED** [review]

File: src/routes/_components/LengthIndicator.html [hunk 3]
```diff
@@ -42,11 +44,12 @@
...
+        // When approaching the limit, notify screen reader users
+        if (lengthToDisplayDeferred > WARN_THRESHOLD) {
```
Comment (nolanlawson) [inline src/routes/_components/LengthIndicator.html:49]:
```suggestion
        if (lengthToDisplayDeferred > WARN_THRESHOLD || lengthToDisplayDeferred <= 0) {
```

@nvdaes It seems to me that we should keep warning if the user reaches 0 or goes beyond the limit (-1, -2, etc.). Let me know what you think of this change. :arrow_up:

**Review by nolanlawson — CHANGES_REQUESTED** [review]
See question above, otherwise LGTM

**Review by nolanlawson — APPROVED** [review]

**Comment by nvdaes** [comment]
Thanks @nolanlawson. Several times I had to rewrite a toot since I exceeded the limit of characters and this may help a lot when we type a long message

**Pull request merged**